lib/src/audio/sounds.dart

```diff
@@ -1,13 +1,13 @@
 List<String> soundTypeToFilename(SfxType type) {
   switch (type) {
-    case SfxType.drawX:
-      return [
+    case SfxType.huhsh:
```

Comment:
Why this change? Just curious.

Reply:
Since the sound can be used for a purpose other than drawing X